setup.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Note: To use the 'upload' functionality of this file, you must:
+#   $ pip install twine
+
+import io
+import os
+import sys
+from shutil import rmtree
+
+from setuptools import find_packages, setup, Command
+
+# Package meta-data.
+NAME = 'pie'
+DESCRIPTION = 'A Framework for Joint Learning of Sequence Labeling Tasks'
+URL = 'https://github.com/emanjavacas/pie'
+EMAIL = 'me@example.com'
+AUTHOR = ' Enrique Manjavacas; Mike Kestemont'
+REQUIRES_PYTHON = '>=3.6.0'
+VERSION = "0.2.0a0"
+
+# What packages are required for this module to be executed?
+REQUIRED = [
```

Comment:
Maybe this can be read from requirements.txt to avoid duplication?

Reply:
I'd be fine with this. I know some people do not like to have requirements.txt read into setup.py. What do you prefer in these two options listed in the answer : https://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py

Reply:
I guess this would be fine: https://stackoverflow.com/a/14399775 but I don't have a real preference, just whatever is easier to maintain later

Reply:
Then I'll do this ;)

Reply:
Done.